test/controllers/api/docs_controller_test.exs

```diff
@@ -58,16 +52,59 @@ defmodule HexWeb.API.DocsControllerTest do
     assert conn.resp_body == "HEYO"
   end
 
+  @tag :integration
```

Comment:
Right. What I was trying to say, is that since the release/overwrite docs logic is already tested in the previous test, it might not make sense to hit S3 to test essentially the same stuff, and using Store.Local adapter should catch logical errors. I guess I'll just leave this tag here.